asset/file_test.go

```
@@ -0,0 +1,311 @@
+		},
+		{
+			name: "File object is nil",
+			file: nil,
+			uuid: "xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx",
+			want: "",
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			result := tt.file.RootPath(tt.uuid)
```

golangci-lint (1.64.8)

loopclosure: loop variable tt captured by func literal

(govet)